docs/commands.md

```diff
@@ -0,0 +1,486 @@
+
+#### Arguments
+
+| Name | Type | Description |
+| :------ | :------ | :------ |
+| `content` | `any` | - |
+| `contentType?` | `any` | - |
+| `label?` | `string` | The content to be set as base64 encoded string. |
+
+#### Returned Result
+
+``null``
+
+### `toggleEnrollTouchId`
+
+`POST` **`/session/:sessionId/appium/simulator/toggle_touch_id_enrollment`**
+
+Toggle whether the device is enrolled in the touch ID program
+
+#### Arguments
+
+| Name | Type | Default value |
+| :------ | :------ | :------ |
+| `enabled?` | `any` | `true` |
+
+#### Returned Result
+
+``null``
+
+### `touchId`
+
+`POST` **`/session/:sessionId/appium/simulator/touch_id`**
+
+Trigger a touch/fingerprint match or match failure
+
+#### Arguments
+
+| Name | Type | Default value |
+| :------ | :------ | :------ |
+| `match` | `any` | `true` |
+
+#### Returned Result
+
+``null``
+
+### `startRecordingScreen`
+
+`POST` **`/session/:sessionId/appium/start_recording_screen`**
+
+Start recording the device screen. This functionality is available in the iOS Simulator since
+Xcode 9, and in real devices since iOS 11.
+
+Screen activity is recorded to an MPEG-4 file. Note that audio is not recorded with the video file.
+If the screen recording has already been started, this command will force stop it and start a new
+recording. The previously recorded video file will also be deleted.
+
+!!! info
+
+    This command requires the `ffmpeg` utility to be installed (`brew install ffmpeg`)
+
+**`Throws`**
+
+If the screen recording has failed to start.
+
+#### Arguments
+
+| Name | Type |
+| :------ | :------ |
+| `options?` | `any` |
+
+#### Returned Result
+
+`string`
+
+Base64-encoded content of the recorded media file if any screen recording is currently running,
+or an empty string.
+
+### `stopRecordingScreen`
+
+`POST` **`/session/:sessionId/appium/stop_recording_screen`**
+
+Stop an ongoing screen recording and return the video. This functionality is available in the iOS
+Simulator since Xcode 9, and in real devices since iOS 11.
+
+
+If no screen recording process is running, the command will attempt to retrieve the most recently
+recorded file. If no previously recorded file is found, the method will return an empty string.
+
+**`Throws`**
+
+If there was an error while getting the name of a media file, or the file content cannot be uploaded
+to the remote location.
+
+#### Arguments
+
+| Name | Type |
+| :------ | :------ |
+| `options?` | `any` |
+
+#### Returned Result
+
+``null`` \| `string`
+
+Base64-encoded content of the recorded media file if `remotePath` parameter is empty or null,
+or an empty string.
+
+### `getSize`
+
+`GET` **`/session/:sessionId/element/:elementId/size`**
+
+Get the size of an element
+
+#### Returned Result
+
+`Size`
+
+The positions of the element
+
+### `submit`
+
+`POST` **`/session/:sessionId/element/:elementId/submit`**
+
+Submit the form an element is in
+
+#### Returned Result
+
+``null``
+
+### `click`
```

Comment:
this method should be marked as deprecated. Such endpoint does not exist in the W3C spec

Reply:
Makes sense, I was unsure about this method since it wasn't marked as deprecated in the current docs. What would be the replacement? A quick code search showed there is `nativeClick` which calls `/session/:sessionId/element/:elementId/click`, would that be it?

Reply:
You can always check the spec if you are unsure: https://www.w3.org/TR/webdriver2

For this particular endpoint https://www.w3.org/TR/webdriver2/#dfn-element-click might not be a good match, because it does not contain an element id. From W3C perspective the best fit could be W3C actions. Or https://github.com/appium/appium-xcuitest-driver/blob/master/docs/execute-methods.md#mobile-tap